proto/configpb.proto

```diff
@@ -12,7 +13,9 @@ option (rustproto.lite_runtime_all) = true;
 service Config {
     rpc Create(CreateRequest) returns (CreateResponse) {}
     rpc Get(GetRequest) returns (GetResponse) {}
-    rpc Update(UpdateRequest) returns (UpdateResponse) {}
+    rpc Update(UpdateRequest) returns (UpdateResponse) {
+        option (google.api.http) = { post: "/pd/api/v1/config" body: "*" };
```

Comment:
```suggestion
        option (google.api.http) = { post: "/config" body: "*" };
```

I think just `/config` is enough, it's better to not leak PD and api version to protobuf definition